lib/chef/provider/git.rb

```diff
@@ -344,6 +344,12 @@ def sha_hash?(string)
         string =~ /^[0-9a-f]{40}$/
       end
 
+      def repo_url
+        if new_resource.sensitive
+          "**Supressed Sensitive URL**"
+        else
+          new_resource.repository
```

Comment:
There's a missing end here that's causing all the specs to fail

Reply:
Thanks @tas50